panel/chat/feed.py

```diff
@@ -699,7 +693,7 @@ def undo(self, count: int = 1) -> List[Any]:
             return []
         messages = self._chat_log.objects
         undone_entries = messages[-count:]
-        self[:] = messages[:-count]
+        self._chat_log.objects[:] = messages[:-count]
```

Comment:
This will not trigger a render, is that intentional?

Reply:
Thanks for catching, no.